nixos/modules/services/misc/n8n.nix

```diff
@@ -25,6 +25,11 @@ in
       description = ''
         Configuration for n8n, see <https://docs.n8n.io/hosting/environment-variables/configuration-methods/>
         for supported values.
+
+        ::: {.warning}
+        Since n8n 1.70.0, the configuration file is deprecated.
+        Use `services.n8n.environment` instead.
+        :::
```

Comment:
Only new users who check the options will know about this change, current n8n users will have no idea about it.
What about using `lib.mkRemovedOptionModule` (example: https://github.com/NixOS/nixpkgs/commit/07abe18272c38d5208a29338f17a011f6ae34ab1#diff-10f8ab842ccf02563cd0a4c7a83ecea92b32ffcd213027ae847ba0b67846d164R21-R25)  instead? It's a little harsher change where users are forced update their config. For a lighter effect you could add a warning for it (example: https://github.com/NixOS/nixpkgs/commit/f2e4cb7f#diff-f6e854fdd6d963d86288a75821f01e727ec4ad2c2c78d3aaf6ddadfea40354e7R49-R51).